### PR TITLE
Use the new from_dataset method to initialize objects in examples. 

### PR DESCRIPTION
**Pull request by arokem**
Follow up on #121.

TODO: 
- [x] Find all the examples that currently load these datasets in a few steps.
- [x] It's nice to have this method, but we will also want there to be an example that shows how to load your own data from CSV files step by step. 
- [x] Mention somewhere that these CSVs are generated and point to pyAFQ and file-format spec.

### CODE REVIEW

File: examples/plot_hbn_site_profiles.py [hunk 1]
```diff
@@ -44,32 +44,30 @@
...
 
-##########################################################################
-# Train / test split
+########################################################################## Train
+# / test split
 # ------------------
 #
-# We can use the dataset in the :func:`train_test_split` function just as we
-# would with an array.
+# We can pass the :class:`AFQDataset` class instance to Sckit Learn's
```
Comment (richford) [inline examples/plot_hbn_site_profiles.py:69]:
```suggestion
# We can pass the :class:`AFQDataset` class instance to scikit-learn's
```

Just to copy how they refer to scikit-learn in their own docs.

**Comment by arokem** [comment]
This is ready for a review.

**Comment by richford** [comment]
There are some flake8 errors on build (3.9), mostly due to unused imports.

File: examples/plot_als_classification.py [hunk 2]
```diff
@@ -27,37 +27,49 @@
...
+# these studies). By passing `"sarica"`, we request that the software download
+# the data from this study and initialize an object for us from this data.
+
+
+afqdata = AFQDataset.from_study("sarica")
 
+# Examine the data
+# ----------------
 # afqdata is a namedtuple. You can access it's fields using dot notation or by
 # unpacking the tuple. To see all of the available fields use `afqdata._fields`
```
Comment (richford) [inline examples/plot_als_classification.py:56]:
```suggestion
# ``afqdata`` is an ``AFQDataset`` object, with properties corresponding to the tractometry features and phenotypic targets.
```

**Review by richford — CHANGES_REQUESTED** [review]
LGTM, with one small comment. There is an explanation that worked well with the `afqdata` `namedtumple` returned by `load_sarica` but not with the `AFQDataset` instance.

**Pull request merged**